### PR TITLE
Some more AppSubUrl fixes

### LINKED ISSUE #8461: Wrong OAuth2 Form target when running on modified root URL
- Gitea version (or commit ref): v1.9.4
- Git version: 2.22.0
- Operating system: gitea/gitea on Docker on Ubuntu 18.04.3 LTS
- Database (use `[x]`):
  - [x] PostgreSQL
  - [ ] MySQL
  - [ ] MSSQL
  - [ ] SQLite
- Can you reproduce the bug at https://try.gitea.io:
  - [ ] Yes (provide example URL)
  - [x] No (because its root matches the server root)
  - [ ] Not relevant
- Log gist: https://gist.github.com/PeterNerlich/275d199733c38d2aee8a607b5d7bf84a

## Description

I have set gitea up in docker behind a reverse proxy (Traefik). We are not given subdomains so we have to work with prefixes like `/git/`. When trying to integrate drone into our setup, granting authorization to it fails. I think I traced it down to the "grant or cancel" form action url leading to `/login/oauth/grant` instead of `/git/login/oauth/grant`. Manipulating the HTML and clicking the button then worked.

## Screenshots

Screenshot from 2019-10-11 11-38-17

### PR DESCRIPTION
**Pull request by zeripath**
Fix #8461 - fix misspelling of {{AppSubUrl}} and other misspelling in template
Fixes /explore and organisation redirection

### CODE REVIEW

**Review by jolheiser — APPROVED** [review]
LGTM, however any reason to not use `path.Join` instead?

**Comment by zeripath** [comment]
@jolheiser Unfortunately, AppSubUrl is `""` when it's empty not `"/"` so `path.Join(...)` wouldn't produce the correct answer.

**Comment by jolheiser** [comment]
🤦‍♂ Of course, I didn't resolve it correctly in my head.

**Review by lafriks — APPROVED** [review]

File: options/locale/locale_en-US.ini [hunk 1]
```diff
@@ -259,7 +259,7 @@ openid_signin_desc = Enter your OpenID URI. For example: https://anne.me, bob.op
 disable_forgot_password_mail = Account recovery is disabled. Please contact your site administrator.
 email_domain_blacklisted = You cannot register with your email address.
 authorize_application = Authorize Application
-authroize_redirect_notice = You will be redirected to %s if you authorize this application.
```
Comment (guillep2k) [inline options/locale/locale_en-US.ini:262]: How does this affect Crowdin?

Comment (zeripath) [reply options/locale/locale_en-US.ini:262]: They'll have to be redone unfortunately but I thought it worth fixing.

Comment (techknowlogick) [reply options/locale/locale_en-US.ini:262]: @zeripath If this gets backported, would could you apply the changes to the other translation files too? I think this is a reasonable PR to backport, but I'll leave that up to you if you have time.

Comment (zeripath) [reply options/locale/locale_en-US.ini:262]: I can drop it if preferred

**Pull request merged**